src/scripts/main.js

```diff
@@ -83,17 +83,15 @@ const views = [].slice.call(document.querySelectorAll('webview'));
 function webViewLoaded() {
 	const currentActive = document.querySelector('webView.active');
 	if (currentActive) kickOutIframe(currentActive);
-	this.classList.remove('buffering');
-	this.classList.add('active');
+	this.setAttribute('class', 'active');
 	this.removeEventListener('load', webViewLoaded);
 	window.dispatchEvent(visibleEvt);
 }
 
 function kickOutIframe(webView) {
-	webView.classList.remove('active');
-	webView.classList.remove('buffering');
-	webView.classList.add('done');
-	setTimeout(() => webView.src = 'about:blank', 500);
```

Comment:
You should still set it to about:blank somewhere lest the page sit in memory or carry on playing audio.